usaspending_api/etl/management/commands/load_submission.py

```diff
@@ -642,11 +642,41 @@ def load_file_c(submission_attributes, db_cursor, award_financial_frame):
             'program_activity': row.get('program_activity'),
         }
 
+        # individual FILE C D linkage
+        if not created and total_rows < 100000:
+            kwargs = {}
+            kwargs['recipient_id__isnull'] = False
+            if row.get('piid') is not None:
```

Comment:
I'm trying to come up with a more elegant way for a series of "if not none then add to dict" steps like this, but I don't want to hold up your PR for it.  I'll let you know later if I come up with one.